src/components/postLayout.js

```diff
@@ -18,7 +19,7 @@ const PostContent = styled.div`
   }
 
   figcaption {
-    color: #828282;
+    color: #6680244;
```

Comment:
A hex code is so called because it should have six characters - these have seven values and are invalid.